README.md

```diff
@@ -40,3 +40,44 @@ Finally, the gateway accepts a custom binary format for data ingestion. The data
 [REST Proxy documentation]: https://docs.confluent.io/current/kafka-rest/api.html
 [RADAR-Auth]: https://github.com/RADAR-base/ManagementPortal/tree/master/radar-auth
 [ManagementPortal]: https://github.com/RADAR-base/ManagementPortal
+
+## Flow
+
+The gateway is a simple component which forwards requests to Kafka. It requires an access token with `MEASUREMENT.CREATE` scope for the `res_gateway` resource. The flow is as follows:
+
+### Initialization
+
+Initialization is limited to startup of the container and loading/validating configurations such as the Kafka configuration and the RADAR-Auth configuration.
+
+```mermaid
+sequenceDiagram
+    participant gateway as RADAR-Gateway
```

Comment:
When it loads and verifies Management Portal and Kafka, should the init not mention these components in the sequence diagram?